code/modules/surgery/rig_removal.dm

```diff
@@ -0,0 +1,58 @@
+//Procedures in this file: Unsealing a Rig.
```

Comment:
@MarsM0nd My original intention was to make any crew rigsuits lack the NODROP flag on their backpack (and possibly assign a high unequip time). They are designed to retract gracefully if they are ripped off of their host.